packages/builders/src/interactions/slashCommands/Assertions.ts

```diff
@@ -35,14 +40,22 @@ export function validateMaxOptionsLength(options: unknown): asserts options is T
 }
 
 export function validateRequiredParameters(
-	name: string,
-	description: string,
+	name: string | undefined,
```

Comment:
`name` and `description` are casted as non-undefined on SlashCommandBuilder, but I figure they should be treated as potentially undefined wherever they can be, since they can be undefined.